CSharp/BatchExplorer/Helpers/BasicDataProvider.cs

```diff
@@ -73,6 +73,21 @@ public Task CreateJobAsync(CreateJobOptions options)
             return poolModels;
         }
 
+        public async Task<IList<CertificateModel>> GetCertificatesCollectionAsync()
+        {
+            IPagedEnumerable<Certificate> certificates = this.Service.ListCertificates(OptionsModel.Instance.ListDetailLevel);
```

Comment:
Sadly I think this will fail since the detail level for other items doesn't really mesh well for certificates.  For example if you go to options and check "use stats during list" I think this will cause the ListCerts call to fail.

For now I would just remove the detailLevel here...?  We really should have a checkbox for each resource type that supports detail level (i.e. use stats for Job, stats for Pool, etc)